editor.planx.uk/src/components/Feedback/MoreInfoFeedback.tsx

```diff
@@ -36,6 +36,16 @@ const MoreInfoFeedbackComponent: React.FC = () => {
   const [currentFeedbackView, setCurrentFeedbackView] =
     useState<View>("yes/no");
   const [feedbackOption, setFeedbackOption] = useState<Sentiment | null>(null);
+  const feedbackComponentRef = useRef<HTMLDivElement | null>(null);
+
+  useEffect(() => {
+    if (currentFeedbackView === "input") {
+      feedbackComponentRef.current?.scrollIntoView({
+        behavior: "smooth",
+        block: "start",
+      });
+    }
+  }, [currentFeedbackView]);
```

Comment:
A custom hook might be a slightly cleaner way of reusing this logic across both components / files?

Something like - 

```ts
const useScrollIntoView = (currentFeedbackView) => {
  const feedbackComponentRef = useRef<HTMLDivElement | null>(null);

  const shouldScrollToView = () => **logic here**
  
  useEffect(() => {
    if (shouldScrollToView()) {
      feedbackComponentRef.current?.scrollIntoView({
        behavior: 'smooth',
        block: 'start',
      });
    }
  }, [currentFeedbackView]);

  return feedbackComponentRef;
};
```

Alternatively just a wrapper for the feedback components with this logic could work also. If this makes things more complex or there's something I'm missing here no sweat!